### PR TITLE
Add UI for facebook login

### PR DESCRIPTION
**Pull request by Prabhakar-Poudel**
Fetch AccessToken from facebook
Create facebook app

### CODE REVIEW

File: client/src/components/login/FacebookButton.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```
Comment (chirag-shinde) [inline client/src/components/login/FacebookButton.js:1]: LGTM

File: client/src/components/login/login.css [hunk 2]
```diff
@@ -8,4 +8,8 @@
     font-size: 30px;
     text-align: center;
     font-weight: bold;
+}
+
+.login-button-container {
+    margin: 20px auto;
```
Comment (chirag-shinde) [inline client/src/components/login/login.css:14]: Can you add a screenshot of how the Login page renders?

File: client/yarn.lock [hunk 3]
```diff
@@ -3287,6 +3287,11 @@ dotenv@6.2.0:
   resolved "https://registry.yarnpkg.com/dotenv/-/dotenv-6.2.0.tgz#941c0410535d942c8becf28d3f357dbd9d476064"
   integrity sha512-HygQCKUBSFl8wKQZBSemMywRWcEDNidvNbjGVyZu3nbZ8qq9ubiPoGLMdRDpfSrpkkm9BXYFkpKxxFX38o/76w==
 
+dotenv@^7.0.0:
```
Comment (chirag-shinde) [inline client/yarn.lock:3290]: Create-react-app comes with the dotenv package in react-scripts, we don't need to add it to dependencies explicitly.

**Review by chirag-shinde — APPROVED** [review]

**Comment by Prabhakar-Poudel** [comment]
fblogin

**Comment by Prabhakar-Poudel** [comment]
Currently the default button does not match well with google button styling.
We will have to fix that.

File: client/yarn.lock (hunk 3 above)
Comment (Prabhakar-Poudel) [reply client/yarn.lock:3290]: Fixed.

File: client/src/components/login/login.css (hunk 2 above)
Comment (Prabhakar-Poudel) [reply client/src/components/login/login.css:14]: Done.

**Pull request merged**